server/routes/visitors.ts

```diff
@@ -413,5 +410,93 @@ export default function routes(
     })
   })
 
+  router.post('/check-your-booking', async (req, res) => {
+    const { visitSessionData } = req.session
+    checkSession({
+      stage: 5,
+      visitSessionData,
+      res,
+    })
+
+    const { offenderNo } = req.session.visitSessionData.prisoner
+
+    const additionalSupport = visitSessionData.additionalSupport?.keys?.map(key => {
```

Comment:
Could move this to utils function as same is used in GET route? Though perhaps not worth it at this stage as we'll probably need change additional support a bit once Phil's back-end stuff done...

Reply:
I did wonder, let's wait for us to pull in Phil's changes and go from there.